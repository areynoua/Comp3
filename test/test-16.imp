begin
@foo(3);
function @foo(i) do
 print(i)
end
end
