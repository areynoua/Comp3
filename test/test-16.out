Call to undefined function:
@foo is not a function name
